appveyor.yml

```diff
@@ -78,7 +78,9 @@ build_script:
       "https://github.com/electron/electron"
   - ps: >-
       if ($env:GN_CONFIG -eq 'release') {
-        gclient sync --with_branch_heads --with_tags --reset
+        if ($(gclient sync --with_branch_heads --with_tags;$LASTEXITCODE -ne 0)) {
+          Write-warning "Error with gclient sync; continuing anyway"
```

Comment:
...uhh? why would we want to continue when gclient sync fails?

Reply:
@nornagon there was some weird exit code that was happening but the sync was actually successful

Reply:
is it still happening? We should figure out what's going on, because this will cause bizarre and hard-to-debug issues later if gclient sync starts legitimately failing.

Reply:
@nornagon yes - unfortunately: https://ci.appveyor.com/project/electron-bot/electron-ia32-testing/builds/29346983

Reply:
looks like `--ignore_locks` is probably the right fix.